programs/drift-competitions/src/instructions/update_switchboard_function.rs

```diff
@@ -0,0 +1,119 @@
+use anchor_lang::prelude::*;
+use anchor_spl::associated_token::AssociatedToken;
+use anchor_spl::token::{Mint, Token};
+use switchboard_solana::prelude::*;
+use crate::signer_seeds::get_function_authority_seeds;
+use crate::state::Competition;
+
+/// The minimum guess that can be submitted, inclusive.
+pub const MIN_RESULT: u32 = 1;
+/// The maximum guess that can be submitted, inclusive.
+pub const MAX_RESULT: u32 = 1_000_000;
```

Comment:
is this just an example or hardcoded upper bound? this bound should be much higher if it even needs to exist at all